ipaserver/plugins/baseuser.py

```diff
@@ -728,3 +739,138 @@ def post_callback(self, ldap, dn, entry_attrs, *keys, **options):
         self.obj.convert_usercertificate_post(entry_attrs, **options)
 
         return dn
+
+
+class ModCertMapData(LDAPModAttribute):
+    attribute = 'ipacertmapdata'
+    takes_options = (
+        DNParam(
+            'issuer?',
+            cli_name='issuer',
+            label=_('Issuer'),
+            doc=_('Issuer of the certificate'),
+            flags=['virtual_attribute', 'no_create', 'no_update']
+        ),
+        DNParam(
+            'subject?',
+            cli_name='subject',
+            label=_('Subject'),
+            doc=_('Subject of the certificate'),
+            flags=['virtual_attribute', 'no_create', 'no_update']
+        ),
+        Bytes(
+            'usercertificate*', validate_certificate,
+            cli_name='certificate',
+            label=_('Certificate'),
+            doc=_('Base-64 encoded user certificate'),
+        ),
+    )
+
+    @staticmethod
+    def _build_mapdata(subject, issuer):
+        return u'X509:{issuer}{subject}'.format(
+            issuer='<I>{}'.format(issuer.x500_text()) if issuer else '',
+            subject='<S>{}'.format(subject.x500_text()) if subject else '',
+            )
```

Comment:
Because it has additional costs for us (in maintenance, testing, ...) for very questionable benefit. Nobody requested it and I seriously doubt anybody would. What does it matter if it's user friendly when there's no user to use it that way?

AD does not support issuer-only AFAICT as it is non-sensical, but it supports [other variants](https://blogs.msdn.microsoft.com/spatdsg/2010/06/18/howto-map-a-user-to-a-certificate-via-all-the-methods-available-in-the-altsecurityidentities-attribute/) which this PR does not implement, so clearly supporting everything AD supports is not the goal here, and I don't think it ever was in the first place.